src/modules/infobox/components/FeatureAreaInfo.vue

```diff
@@ -34,7 +34,7 @@ const humanReadableArea = computed(() => {
     const calculatedArea = area(transformIntoTurfEquivalent(geometryWgs84.value))
     let result = ''
     if (calculatedArea) {
-        result += roundValueIfGreaterThan(calculatedArea, 1000, 1000000)
+        result += roundValueIfGreaterThan(calculatedArea, 10000, 1000000)
         if (calculatedArea > 10000) {
             result += ' km'
         } else {
```

Comment:
For better code readability and to be bullet proof, the constant should be defined as constant instead of hardcoding. If I understand correctly the threshold should be also used to define the unit ?

Should this not be something like this ?

```javascript
const humanReadableArea = computed(() => {
    const calculatedArea = area(transformIntoTurfEquivalent(geometryWgs84.value))
    let result = ''
    if (calculatedArea) {
        const divider = 1000000
        const threshold = 10000
        result = roundValueIfGreaterThan(calculatedArea, threshold, divider)
        if (calculatedArea > threshold) {
            result += ' km'
        } else {        
            result += ' m'
        }
    }
    return result
})